### PR TITLE
Prevent outputting converted JSON twice

### PR DESCRIPTION
**Pull request by clenk**
Also, fix help message for using validator args.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -120,7 +120,7 @@ STIX 1.1.1 - 1.2.1 content to STIX 2.0 content::
       --validator-args VALIDATOR_ARGS
                             Arguments to pass to stix-validator. Default: --strict-
                             types Example: stix2_elevator.py <file> --validator-
-                            args "-v --strict-types -d 212"
+                            args="-v --strict-types -d 212"
```
Comment (gtback) [inline README.rst]: Is `=` actually required or is it more of a consistency thing?

Comment (clenk) [reply README.rst]: It wouldn't work for me in zsh without the `=`.

Comment (emmanvg) [reply README.rst]: I was able to test it using `=` without any problems on Windows command-line. The behavior is still correct.

**Comment by clenk** [comment]
Corerction: In zsh it only gave me an error when I only passed one argument to `--validator-args`. If I passed multiple, it worked okay. If I added a space before or after the single validator argument I wanted to use, it worked okay.

**Comment by gtback** [comment]
Weird. It seems like ZSH will remove quotes around any argument that doesn't need them (like `"-v"`), and as a result Python treats that as a different argument to the elevator, rather than the argument to the elevator. In any case, I'm OK with the "=".

**Pull request merged**